src/pyfaf/actions/cleanup_packages.py

```diff
@@ -112,12 +106,15 @@ def run(self, cmdline, db):
                         .filter(Package.build_id == build.build_id)
                         .all()):
                 self.delete_package(pkg, cmdline.dry_run)
-                self.delete_package(pkg)
         return 0
 
     def tweak_cmdline_parser(self, parser):
-        opsys_group = parser.add_argument_group("Required together")
-        opsys_group.add_opsys(positional=True, helpstr="operating system")
-        opsys_group.add_opsys_release(positional=True, helpstr="release")
-        arch_group = parser.add_argument_group("Individual")
-        arch_group.add_arch(helpstr="architecture")
+        #opsys_group = parser.add_argument_group("Required together")
+        #opsys_group.add_argument("OPSYS", nargs='?', help="operating system")
+        #opsys_group.add_argument("RELEASE", nargs='?', help="release")
+        #arch_group = parser.add_argument_group("Individual")
+        #arch_group.add_argument("--arch", help="architecture")
```

Comment:
Remove the commented code, please.

Reply:
Removed. I left them in on purpose to remind anybody working on this in the future to implement the arg groups and their mutual exclusiveness on the wtforms level, but oh well.

Reply:
That really never works, same as TODO comments. Better create a new issue for it with detailed information.